astropy/io/ascii/tests/test_read.py

```diff
@@ -704,13 +705,13 @@ def test_default_missing(fast_reader):
     assert dat["a"].dtype.kind == "i"
 
     # Same test with a fixed width reader
-    table = "\n".join(
-        [
-            " a   b   c   d ",
-            "--- --- --- ---",
-            "  1   3        ",
-            "  2     4.0  ss",
-        ]
+    table = dedent(
+        """\
+         a   b   c   d
+        --- --- --- ---
+          1   3
```

Comment:
Hmm, this actually is not the same test any more, since the trailing spaces are removed. I think the original is a clearer in its intent.

Reply:
Done.